ProjectFinderApi/V1/Gateways/UsersGateway.cs

```diff
@@ -29,5 +30,10 @@ public User CreateUser(CreateUserRequest createUserRequest)
             _databaseContext.SaveChanges();
             return user;
         }
+
+        public IEnumerable<User> GetUsers()
```

Comment:
I know you'll be adding paging in a separate PR, so this can stay as it is for now.